e2e/sidebar.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { expect, test } from "./_support/fixtures";
+import { LecturePage } from "./page-objects/lecture_page";
+
+test("can access tutorial submission page (only as tutor)",
```

Comment:
Note that such a test is necessary since other tests go directly to the respective page (e.g. `/lectures/:id/tutorials`), instead of using the sidebar links. This is good for other tests, but at one point we want to also test that the sidebar works as expected.